Website/admin/Menus/ModuleActions/ModuleActions.ascx.cs

```diff
@@ -81,6 +87,7 @@ protected override void OnInit(EventArgs e)
             actionButton.Click += actionButton_Click;
 
             ClientResourceManager.RegisterStyleSheet(Page, "~/admin/menus/ModuleActions/ModuleActions.css", FileOrder.Css.ModuleCss);
+            ClientResourceManager.RegisterStyleSheet(Page, "https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css", FileOrder.Css.ModuleCss);
```

Comment:
At this stage of development of this feature - we have not decided for sure whether to use Image Sprites (as before) or to switch to FA icons (which we have used in the new DCC Manager).